tests/tox.sh

```diff
@@ -89,14 +89,14 @@ CURRENT_CEPH_STABLE_RELEASE="$(echo $FLAVOR|awk -F ',' '{ print $1}')"
 # so we will build the desired CEPH_STABLE_RELEASE since the current patch didn't change the ceph version
 # Since we test all the Ceph releases, we will always test the impacted one
 if [[ "$CEPH_STABLE_RELEASE" != "$CURRENT_CEPH_STABLE_RELEASE" ]]; then
-  FLAVOR="$CEPH_STABLE_RELEASE,x86_64,ubuntu,16.04,_,ubuntu,16.04"
+  FLAVOR="$CEPH_STABLE_RELEASE,ubuntu,16.04"
 fi
 
 echo "Building flavor $FLAVOR"
 sudo make FLAVORS="$FLAVOR" build.parallel
 
 current_branch="$(git rev-parse --abbrev-ref HEAD)"
-# build tag from luminous,x86_64,centos,7,_,centos,7 to luminous-centos-7-x86_64
+# build tag from luminous,centos,7 to luminous-centos-7-x86_64
```

Comment:
Since ` x86_64` is implicit you need to force it by detecting it.
So something like this should work:

```
TAG=$(echo "$FLAVOR"|awk -F ',' '{ print $1"-"$2"-"$3 }')-$(arch)
```

Reply:
@BlaineEXE @leseb does this item have been handled ?